src/components/existing-request/existing-request-display.vue

```diff
@@ -768,4 +780,10 @@ export default class ExistingRequestDisplay extends Mixins(
     text-decoration: none;
   }
 }
+
+.warning-message {
+  color: $gray7;
+  background-color: rgba(211, 39, 44, .1);
```

Comment:
Let's keep with best practices and create a CSS variable for this colour (in theme.scss).

Reply:
Thanks!